### PR TITLE
Add gcc 11.2.0 and libssp 11.2.0

### PR DESCRIPTION
**Pull request by satmandu**
- Bugfix release
- Should be tested on all architectures

Works properly:
- [x] x86_64
- [x] i686 (? Needs testing)
- [x] armv7l (needs testing on actual hardware.) @uberhacker Could you do that?

### CODE REVIEW

**Comment by uberhacker** [comment]
On armv7l, I get:
```
$ gcc --version
gcc: /usr/local/lib/libc.so.6: version `GLIBC_2.28' not found (required by gcc)
```
On i686, I get:
```
$ gcc --version
Segmentation fault (core dumped)
```

**Comment by satmandu** [comment]
Thanks for that... I'll do a rebuild on armv7l...

**Comment by satmandu** [comment]
armv7l should be fixed here.

**Comment by satmandu** [comment]
This should be ready to have i686 added. @uberhacker If you could test this newer build on armv7l when you have time too? It should work with pre-glibc 2.32 now.

File: packages/gcc11.rb [hunk 1]
```diff
@@ -4,28 +4,27 @@
...
+  source_sha256 'd08edc536b54c372a1010ff6619dd274c0f1603aa49212ba20f7aa2cda36fa8b'
 
   binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gcc11/11.1.0-2_armv7l/gcc11-11.1.0-2-chromeos-armv7l.tpxz',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gcc11/11.1.0-2_armv7l/gcc11-11.1.0-2-chromeos-armv7l.tpxz',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gcc11/11.1.0-2_i686/gcc11-11.1.0-2-chromeos-i686.tpxz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gcc11/11.1.0-2_x86_64/gcc11-11.1.0-2-chromeos-x86_64.tpxz'
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gcc11/11.2.0_armv7l/gcc11-11.2.0-chromeos-armv7l.tpxz',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gcc11/11.2.0_armv7l/gcc11-11.2.0-chromeos-armv7l.tpxz',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gcc11/11.2.0-1_i686/gcc11-11.2.0-1-chromeos-i686.tar.xz',
```
Comment (uberhacker) [inline packages/gcc11.rb:16]: What is with the `-1`?

Comment (satmandu) [reply packages/gcc11.rb]: Artifact of doing a local rebuild on i686. Fixed.

**Comment by uberhacker** [comment]
Binaries work on both armv7l and i686.  Nice!

**Pull request merged**